man/systemd.service.xml

```diff
@@ -1088,6 +1088,279 @@ ExecStart=/bin/echo $ONE $TWO $THREE</programlisting>
+            <entry>Failed to reset a program's signal mask.</entry>
+          </row>
+          <row>
+            <entry>208</entry>
+            <entry>EXIT_STDIN</entry>
+            <entry>Failed to set up a program's standard input.</entry>
+          </row>
+          <row>
+            <entry>209</entry>
+            <entry>EXIT_STDOUT</entry>
+            <entry>Failed to set up a program's standard output.</entry>
+          </row>
+          <row>
+            <entry>210</entry>
+            <entry>EXIT_CHROOT</entry>
+            <entry>The chroot(2) call failed.</entry>
+          </row>
+          <row>
+            <entry>211</entry>
+            <entry>EXIT_IOPRIO</entry>
+            <entry>Failed to set up a program's IO scheduler priority.</entry>
+          </row>
+          <row>
+            <entry>212</entry>
+            <entry>EXIT_TIMERSLACK</entry>
+            <entry>Failed to set up a program's timer slack.</entry>
+          </row>
+          <row>
+            <entry>213</entry>
+            <entry>EXIT_SECUREBITS</entry>
+            <entry>Failed to set up a program's secure bits.</entry>
+          </row>
+          <row>
+            <entry>214</entry>
+            <entry>EXIT_SETSCHEDULER</entry>
+            <entry>The sched_setscheduler(2) system call failed.</entry>
+          </row>
+          <row>
+            <entry>215</entry>
+            <entry>EXIT_CPUAFFINITY</entry>
+            <entry>The sched_setaffinity(2) system call failed.</entry>
+          </row>
+          <row>
+            <entry>216</entry>
+            <entry>EXIT_GROUP</entry>
+            <entry>Failed to determine group credentials or supplementary groups.</entry>
+          </row>
+          <row>
+            <entry>217</entry>
+            <entry>EXIT_USER</entry>
+            <entry>Setting up the user namespace failed.</entry>
+          </row>
+          <row>
+            <entry>218</entry>
+            <entry>EXIT_CAPABILITIES</entry>
+            <entry>Failed to apply ambient capabilities.</entry>
+          </row>
+          <row>
+            <entry>219</entry>
+            <entry>EXIT_CGROUP</entry>
+            <entry>Setting up the service cgroup failed.</entry>
+          </row>
+          <row>
+            <entry>220</entry>
+            <entry>EXIT_SETSID</entry>
+            <entry>Running the program in a new session failed.</entry>
+          </row>
+          <row>
+            <entry>221</entry>
+            <entry>EXIT_CONFIRM</entry>
+            <entry>Execution of a program has not been confirmed.</entry>
+          </row>
+          <row>
+            <entry>222</entry>
+            <entry>EXIT_STDERR</entry>
+            <entry>Failed to set up a program's standard error output.</entry>
+          </row>
+          <row>
+            <entry>233</entry>
+            <entry>_EXIT_RESERVED</entry>
+            <entry>This used to be connected with tcpwrap. It's not to be reused anymore.</entry>
```

Comment:
We shouldn't talk about services which we don't use. I think you should just comment out that table line in the xml, so that it's easier to see when editing the source, but it does not show up when reading the man page.